masonry_core/src/testing/screenshots.rs

```diff
@@ -5,7 +5,9 @@
 
 use image::{GenericImageView as _, Pixel as _, Rgb, RgbImage};
 
-#[cfg(docsrs)]
+// TODO - Re-enable this once we find a way to load screenshots that doesn't go against our
+// storage quotas.
+#[cfg(FALSE)]
```

Comment:
https://github.com/rust-lang/rust/pull/138632

`cfg(false)` should be stable in 6 weeks!